packages/usa-in-page-navigation/src/index.js

```diff
@@ -98,6 +131,23 @@ const handleScrollToSection = (el) => {
     top: el.offsetTop - inPageNavScrollOffset,
     block: "start",
   });
+
+  if (window.location.hash.slice(1) !== el.id) {
+    window.history.pushState(null, "", `#${el.id}`);
+  }
```

Comment:
Nice feature! I was able to go forwards/back on previously clicked in-page nav items.

Reply:
Yeah this was the trick to solve [the issue you identified earlier](https://github.com/uswds/uswds/pull/5068#pullrequestreview-1271308864), since assigning the hash fragment this way appears to work the same as `window.location.hash = el.id` with the exception of scrolling, which is what we wanted to avoid.